mod/ckeditor/views.php

```diff
@@ -2,7 +2,6 @@
 
 return [
 	'default' => [
-		'ckeditor.js' => __DIR__ . '/vendors/ckeditor/ckeditor.js',
```

Comment:
I see. Though it might not be worth the effort for me updating this PR request if the additional definition of dependencies in require() in init.php to "fix" the issue (even if only as a workaround) is a no-go anyway. And if the real issue is not with the module dependencies but with caching it might not be necessary to change anything in ckeditor plugin code anyway.